cli/Cargo.toml

```diff
@@ -38,6 +39,9 @@ open = "4.0.0"
 phylum_lockfile = { path = "../lockfile", features = ["generator"] }
 phylum_project = { path = "../phylum_project" }
 phylum_types = { git = "https://github.com/phylum-dev/phylum-types", branch = "development" }
+vuln-reach = { git = "https://github.com/phylum-dev/vuln-reach", optional = true }
+vulnreach_types = { path = "../vulnreach_types", optional = true }
+jsonwebtoken = { version = "8.3.0", optional = true }
```

Comment:
Nothing in the current code uses this crate (unless I've missed something). This should be removed.